### PR TITLE
fix: fixed integration tests and selenium version

### PR DESCRIPTION
**Pull request by Razzwan**

### CODE REVIEW

File: test/integration/proposal-tokenTrade.e2e.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import * as uuid from "uuid";
-import { LATEST_ARC_VERSION, hideCookieAcceptWindow, ITestAddresses, gotoDaoPlugins } from "./utils";
+import { LATEST_ARC_VERSION, hideCookieAcceptWindow, gotoDaoPlugins } from "./utils";
 
 describe("Token Trade Proposals", () => {
   let daoAddress: string;
-  let addresses: ITestAddresses;
+  let addresses: any;
```
Comment (roienatan) [inline test/integration/proposal-tokenTrade.e2e.ts:6]: Why this change?

File: package.json [hunk 2]
```diff
@@ -94,7 +94,7 @@
     "test:unit": "jest"
   },
   "dependencies": {
-    "3box": "^1.22.0",
+    "3box": "1.20.2",
```
Comment (orenyodfat) [inline package.json]: why this is downgrade ?

File: test/integration/tsconfig.json [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
-    "lib": ["es2015.promise", "es2016", "es2017", "dom", "esnext.asynciterable"],
-    "module":"commonjs",
+    "outDir": "./.dist/",
+    "sourceMap": false,
+    "target": "es6",
+    "module": "commonjs",
     "moduleResolution": "node",
-    "declaration": true,
-    "declarationDir": "../dist/types",
+    "removeComments": true,
```
Comment (orenyodfat) [inline test/integration/tsconfig.json]: this reduce bundle size?

File: test/integration/proposal-tokenTrade.e2e.ts (hunk 1 above)
Comment (Razzwan) [reply test/integration/proposal-tokenTrade.e2e.ts:6]: @roienatan because interface is incorrect and it lead typescript checks issue. Proper way is just fix interface I gues...

File: package.json (hunk 2 above)
Comment (Razzwan) [reply package.json]: As I understand, versions conflicts with other newest packages. This is the easyest way to fix that conflict. But it can be also a problem of my local environment. I will also task it with latest version of 3box on Travis right now

File: test/integration/tsconfig.json (hunk 3 above)
Comment (Razzwan) [reply test/integration/tsconfig.json]: @orenyodfat hm... actually no. Bundle doesn't contain integration tests. I just copied this config from documentation. I can remove it. It is redundant and doesn't  do anything useful

**Comment by roienatan** [comment]
Is it normal that there is one check here? (instead of two checks)

File: package.json (hunk 2 above)
Comment (Razzwan) [reply package.json]: @orenyodfat when I try to install newest version it even not allow me to finish instalation. Just only this version works properly. Do you know, how to update it correctly?

File: test/integration/proposal-tokenTrade.e2e.ts (hunk 1 above)
Comment (Razzwan) [reply test/integration/proposal-tokenTrade.e2e.ts:6]: @roienatan trying to fix, but interface is deeply different. Code works properly, but we don't have correct interface for it

File: package.json (hunk 2 above)
Comment (orenyodfat) [reply package.json]: not sure.. it works ok with 1.22.0 before.

Comment (Razzwan) [reply package.json]: So, I think, that it is conflict of packages. `3box` and `@wdio` I guess, but not sure

Comment (Razzwan) [reply package.json]: @orenyodfat on my local machine it doesn't even allow me to install that package. Do you know anything about this library? Why do we need newest version?

Comment (Razzwan) [reply package.json]: @orenyodfat I see, that it should improve performance: https://github.com/3box/3box-js/releases . Looks like it require additional settings on the local machine. Perhapse, I have incorrect JDK version installed locally or something similar

Comment (Razzwan) [reply package.json]: @orenyodfat added [issue](https://github.com/3box/3box-js/issues/932) to the project

**Comment by Razzwan** [comment]
> Is it normal that there is one check here? (instead of two checks)

It is because of no branch exists at the same repo. I created PR from fork, because I had no access to this repo. So, just only PR checked, no branch. Actually, we should rather skip second check for all our requests, because one of them is redundant

File: package.json (hunk 2 above)
Comment (Razzwan) [reply package.json]: @orenyodfat fixed rom linux machine

File: test/integration/tsconfig.json (hunk 3 above)
Comment (Razzwan) [reply test/integration/tsconfig.json]: @orenyodfat removed because of redundant

**Review by orenyodfat — APPROVED** [review]
